packages/browser/qa/__tests__/destinations.test.ts

```diff
@@ -3,15 +3,22 @@ import { reportMetrics } from '../lib/benchmark'
 import { browser } from '../lib/browser'
 import { run } from '../lib/runner'
 import { server } from '../lib/server'
-
 jest.setTimeout(100000)
 
+if (!process.env.QA_SAMPLES) {
+  throw new Error('no process.env.QA_SAMPLES')
+}
```

Comment:
Had to fix all the typescript errors now that the normal (base editor) typescript compiler was actually including them.